kolibri/core/assets/src/state/modules/snackbar.js

```diff
@@ -21,6 +21,8 @@ export default {
       state.options = {};
       // set new options
       state.isVisible = true;
+      // options include text, autoDismiss, duration, actionText, actionCallback,
+      // hideCallback
```

Comment:
I know this pre-dated your PR, but just noting that on line 4 only `text` and `autoDismiss` are mentioned. The other ones seem to be added dynamically and are only implicitly part of the API.

I wonder if there's a need to nest these options in their own `options` object? Perhaps the state here should be flattened to

```javascript
  state: {
    isVisible: false,
    text: '',
    autoDismiss: true,
    duration: null,
    actionText: null,
    actionCallback: null,
  },
```

thoughts?